Scarb.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "protocol_metadata"
+version = "0.1.0"
```

Comment:
What about to name it only `metadata` since it could be used by something else than the protocol? (also for the repo name)